accounts/managers.py

```diff
@@ -22,12 +23,14 @@ def withdrawal(self, account, data):
         return self._deposit_or_withdrawal(account, data, TradeLog.TrageCodeChoice.WITHDRAW)
 
     def _deposit_or_withdrawal(self, account, data, code):
-        # todo:에러 처리
-        if code not in TradeLog.TrageCodeChoice.choices:
-            raise APIException(detail="fdfdf")
-        # todo: data params validation
         amount = data.get('amount')
         description = data.get('description')
+        if amount is None or description is None:
```

Comment:
```suggestion
        if not (amount or description):
```
제 생각에는  None은 False 이기 때문에, is None 대신 다음과 같이 not을 이용해서 써도 괜찮을 것 같습니다.

Reply:
차이는 아무값도 안넣는것만 막느냐
false로 처리되는 값만 막느냐인데
예를 들어 description=''로 들어오면
None은 막아내지 않고
not으로 처리하면 막아냅니다
''값을 못들어오게 할까요?

Reply:
'' << 이걸 생각 못했네요!